consensus/beacon/consensus.go

```diff
@@ -360,6 +360,26 @@ func (beacon *Beacon) Finalize(chain consensus.ChainHeaderReader, header *types.
 	// No block reward which is issued by consensus layer instead.
 }
 
+func loadKeysAndProve(tree state.Trie, keys [][]byte, kvs map[string][]byte) (*verkle.VerkleProof, verkle.StateDiff, error) {
+	if vtr, ok := tree.(*trie.VerkleTrie); ok {
+		for _, key := range keys {
+			// XXX workaround - there is a problem in the witness creation
```

Comment:
that is probably not necessary for the post-trie as all values should be loaded in ram at the time that the block is finalized